app/assets/javascripts/misc/previous-address.js

```diff
@@ -0,0 +1,37 @@
+import 'classlist.js';
+
+const I18n = window.LoginGov.I18n;
+
+function previousAddress() {
+  const accordion = document.querySelector('.accordion');
+
+  if (accordion) {
+    const controls = accordion.querySelector('[aria-controls]');
+    const selects = accordion.querySelectorAll('select');
+    const inputs = accordion.querySelectorAll('input');
+
+    controls.classList.remove('display-none');
+
+    const originalHeading = controls.textContent;
+
+    controls.addEventListener('click', function() {
+      const expandedState = accordion.getAttribute('aria-expanded');
+
+      if (expandedState === 'false') {
+        controls.innerHTML = originalHeading;
+
+        [].slice.call(inputs).forEach((input) => {
```

Comment:
In this case I don't want to reset the entire form, just the inputs associated with the Previous Address.  I don't think there is a way to use `reset` with just a specific section of the form is there?